traffic_control/go.mod

```diff
@@ -0,0 +1,13 @@
+module github.com/kubedl/istio_less_ingress_controller
+
+go 1.15
+
+require (
```

Comment:
should the traffic_control package located under pkg/,  instead of the root? @SimonCqk